pkg/istioversion/version.go

```diff
@@ -139,3 +139,27 @@ func mustParseVersionsYaml(yamlBytes []byte) (
 
 	return list, defaultVersion, oldVersion, newVersion, versionMap, aliasList
 }
+
+// GetLatestPatchVersions returns the latest patch versions for all the Major.Minor versions
+func GetLatestPatchVersions() map[string]VersionInfo {
+	latestPatchVersions := make(map[string]VersionInfo)
+	for _, version := range List {
+		majorMinorVersion := fmt.Sprintf("%d.%d", version.Version.Major(), version.Version.Minor())
```

Comment:
Why use `Sprintf`? Isn't the following simpler (and much easier to read)?
```suggestion
		majorMinorVersion := version.Version.Major() + "." + version.Version.Minor()
```

Also, introducing `ver := version.Version` would make the function less verbose.